website/profile/views.py

```diff
@@ -299,6 +312,9 @@ def user_account_password(auth, **kwargs):
     else:
         push_status_message('Password updated successfully.', kind='success', trust=False)
 
+    user.change_password_last_attempt = timezone.now()
```

Comment:
Shuldn't this only be set in `OSFUser.change_password` if the attempt was incorrect?